elixir/apps/web/lib/web/live/clients/index.ex

```diff
@@ -5,7 +5,7 @@ defmodule Web.Clients.Index do
 
   def mount(_params, _session, socket) do
     with {:ok, clients} <- Clients.list_clients(socket.assigns.subject, preload: :actor) do
-      clients = Enum.sort_by(clients, & &1.online?, :desc)
+      clients = Enum.sort_by(clients, & &1.last_seen_at, :desc)
```

Comment:
If we sort by last_seen it should be done in the DB query, we only did it for `online?` because it's stored in a process state: https://github.com/firezone/firezone/blob/main/elixir/apps/domain/lib/domain/clients.ex#L84

```
# at top of file
require Ecto.Query

# after that line
|> Ecto.Query.order_by([clients: clients], desc: clients.last_seen_at, desc: clients.id)
```